### ISSUE TITLE
Put items in craft grid even if there not all there

### ISSUE THREAD

**Issue by Dijkstra1-zz** [mezz/JustEnoughItems#275]
I do like how JEI show when do not have all of the items you need, but sometimes that not all ways right with items with meta/NBT data, and have to one item manually would better ten fill all of grid, maybe shift click the plus button twice would do this, that way you still get see what is missing.

**Comment by xChitose** [comment]
Experiencing the same little issue with an item that has multiple uses.

**Comment by adamk33n3r** [comment]
I would even just like this to put the items that I DO have into the crafting square. Then I could go get the other items I need and come back to finish it.

**Comment by Banderi** [comment]
Yeah, I agree. This was an especially good thing in NEI for crafting interfaces that keep your items in place like the TC crafting table; huge QoL improvement that avoided having to manually transfer all of the items or hold them all off in your inventory - particularly now that TC tables have an overworld stored items rendering, so you can easily see what you put in that table without having to open it. Even more unfortunately, since Mezz said he disliked it, we don't have any transparent overlay showing where the items in the recipe go, so you'd have to remember the recipe in order to manually transfer all the items (another thing that in fact was actually a good ting, albeit having it customizable or disable-able would have been better, but arguably far from being better off not having at all).
Please make this happen!

**Comment by MentalMouse** [comment]
Yeah, this is an issue with ingredients like "any potion" and "any enchanted book", but especially with Blood Magic's orbs (which are used in crafting grids a lot)!

**Comment by Phoupraw** [comment]
Same [here](https://github.com/mekanism/Mekanism/issues/7962).
